### PR TITLE
Ability to choose the Pantheon organization during bootstrapping

### PR DESCRIPTION
**Pull request by AronNovak**
#656

### CODE REVIEW

**Comment by AronNovak** [comment]
```
 [notice] Logged in via machine token.
 [Exec] Done in 0.634s
 [Exec] Running terminus org:list --format=json
 [Exec] Done in 0.433s

 Select a Pantheon organization::
  [13532bc5-3a16-44b6-99c4-abc] Org1
  [4aba616c-cbd2-4dc3-abec-def] Gizra - Org1
  [7c901be9-46f6-4b09-93b3-tgh] Gizra - Org2
```

File: robo-components/BootstrapTrait.php [hunk 1]
```diff
@@ -187,18 +189,27 @@ protected function prepareGithubRepository(string $project_name, string $organiz
...
+    $organizations = $this->taskExec("terminus org:list --format=json")
+      ->printOutput(FALSE)
+      ->run()
+      ->getMessage();
+    $organizations = json_decode($organizations, TRUE);
+
+    // Prompt the user to select an organization.
+    $io = new ConsoleIO($this->input(), $this->output());
+    $organization_choices = array_combine(array_column($organizations, 'id'), array_column($organizations, 'label'));
+    $selected_organization_id = $io->choice('Select a Pantheon organization:', $organization_choices);
```
Comment (AronNovak) [inline robo-components/BootstrapTrait.php:210]:
```suggestion
    $selected_organization_id = $io->choice('Select a Pantheon organization', $organization_choices);
```

File: robo-components/BootstrapTrait.php [hunk 2]
```diff
@@ -187,18 +189,27 @@ protected function prepareGithubRepository(string $project_name, string $organiz
...
-    $result = $this->taskExec("terminus site:create $project_machine_name \"$project_name\" \"bde48795-b16d-443f-af01-8b1790caa1af\" --org=\"$organization\"")
+    $organizations = $this->taskExec("terminus org:list --format=json")
+      ->printOutput(FALSE)
+      ->run()
+      ->getMessage();
+    $organizations = json_decode($organizations, TRUE);
+
+    // Prompt the user to select an organization.
+    $io = new ConsoleIO($this->input(), $this->output());
+    $organization_choices = array_combine(array_column($organizations, 'id'), array_column($organizations, 'label'));
```
Comment (amitaibu) [inline robo-components/BootstrapTrait.php:209]: Maybe if there's a single org don't ask?

File: robo-components/BootstrapTrait.php [hunk 3]
```diff
@@ -187,18 +189,27 @@ protected function prepareGithubRepository(string $project_name, string $organiz
...
+    $selected_organization_id = $io->choice('Select a Pantheon organization', $organization_choices);
+
+    $result = $this->taskExec("terminus site:create $project_machine_name \"$project_name\" \"bde48795-b16d-443f-af01-8b1790caa1af\" --org=\"$selected_organization_id\"")
```
Comment (amitaibu) [inline robo-components/BootstrapTrait.php:212]: Worth a comment what the `bde48795-b16d-443f-af01-8b1790caa1af` is

**Review by amitaibu — COMMENTED** [review]
Thanks

File: robo-components/BootstrapTrait.php (hunk 3 above)
Comment (AronNovak) [reply robo-components/BootstrapTrait.php]: added a comment for that

File: robo-components/BootstrapTrait.php (hunk 2 above)
Comment (AronNovak) [reply robo-components/BootstrapTrait.php:209]: Yep, and when no org, we can throw an exception.

**Comment by AronNovak** [comment]
Ready to go?

**Review by amitaibu — APPROVED** [review]
Thanks

**Pull request merged**